extract/text.py

```diff
@@ -0,0 +1,32 @@
+from .bbox import BBoxFactoryMixin
+
+
+class Text(BBoxFactoryMixin, object):
+    """A class containing information from a <text> node"""
+    attr = {}
+
+    contents = ''
+
+    def __init__(self):
+        self.attr = {}
+        self.contents = ''
+
+
+def get_text_from_xml_element(xml_element):
```

Comment:
I don't really want the constructor to be specific to the xml element, so I've made like a function constructor